### DISCUSSION TITLE
Livestream Idea: Learning Vue

### DISCUSSION THREAD

**Discussion by BOLT04** [EddieHubCommunity/support#198] [Ideas]
I believe there was a livestream about Tailwind CSS, and focused on learning how to use it (something completely new to Eddie). @stemount had the incredible idea of doing the same for Vue. So we could even build something with both 😄 

**Original idea**:
![imagem](https://user-images.githubusercontent.com/18630253/93264449-0bea8100-f79f-11ea-96ca-84d17947be40.png)

**Comment by eddiejaoude** [comment]
I like that idea 👍 .

I do have some Vue related live streams coming up over the next 2 weeks:

- Debbie on NuxtJS https://twitter.com/debs_obrien
- Gwen on VueJS https://twitter.com/faradayacademy

And

- Tan on Svelte https://twitter.com/lihautan

But we can do our own live streams too before / after the above 🤓
